zerver/management/commands/realm_emoji.py

```diff
@@ -49,11 +52,15 @@ def handle(self, *args: Any, **options: str) -> None:
             sys.exit(1)
 
         if options["op"] == "add":
-            img_url = options['img_url']
-            if img_url is None:
+            img_path = options['img_path']
+            if img_path is None:
                 self.print_help("./manage.py", "realm_emoji")
                 sys.exit(1)
-            check_add_realm_emoji(realm, name, img_url)
+            emoji_filename = get_emoji_file_name(os.path.basename(img_path), name)
+            realm_emoji = check_add_realm_emoji(realm, name, emoji_filename)
+            with open(img_path, 'rb') as fp:
+                upload_backend.upload_emoji_image(
+                    fp, emoji_filename, realm.get_admin_users().first())
```

Comment:
`upload_backend` requires an `user_profile` while uploading an image file, so I am using first admin user for that purpose. Not sure if it is completely correct or not but can't think of any other way. Does it look correct to you?

Reply:
Yeah, this is correct, though I think we might want to consider just deleting this management command; it dates from before there was a UI for adding/removing realm emoji at all :)

Reply:
Also, shouldn't this use `add_realm_emoji_and_notify` to notify clients?

Structurally, I think it'd be better for `check_add_realm_emoiji` to handle the filename gymnastics to avoid this sort of bug; is there a reason you split that function?